From 9beff7dbffbada5fa080b66a577fe4bdcb8f0e8a Mon Sep 17 00:00:00 2001
Message-Id: <9beff7dbffbada5fa080b66a577fe4bdcb8f0e8a.1548085556.git.wallinux@gmail.com>
From: Anders Wallin <wallinux@gmail.com>
Date: Mon, 21 Jan 2019 16:45:42 +0100
Subject: [PATCH] perl: Including "." in INC

This patch adds '.' to the search path for perl modules

"The perl binary includes a default set of paths in @INC. Historically it
has also included the current directory (".") as the final entry. While
convenient, this has security implications: for example, where a script
attempts to load an optional module when its current directory is untrusted
(such as /tmp), it could load and execute code from under that directory.

Starting from perl v5.26, "." is always removed by default!"

Signed-off-by: Anders Wallin <wallinux@gmail.com>
---
 perl/Makefile.PL | 1 +
 1 file changed, 1 insertion(+)

diff --git a/perl/Makefile.PL b/perl/Makefile.PL
index cb7039bf9..8ef52a635 100644
--- a/perl/Makefile.PL
+++ b/perl/Makefile.PL
@@ -1,3 +1,4 @@
+use lib '.';
 use strict;
 use warnings;
 use ExtUtils::MakeMaker;
-- 
2.20.1

